### PR TITLE
fix(databend):get data type error

### PR DESCRIPTION
**Pull request by hanyisong**

### CODE REVIEW

File: src/sqlancer/databend/DatabendSchema.java [hunk 1]
```diff
@@ -339,6 +339,13 @@ private static List<DatabendColumn> getTableColumns(SQLConnection con, String ta
                     while (rs.next()) {
                         String columnName = rs.getString("column_name");
                         String dataType = rs.getString("data_type");
+                        if (dataType.contains("NULL")) {
+                            int idx = 0;
+                            while (idx < dataType.length() && dataType.charAt(idx) != ' ') {
```
Comment (mrigger) [inline src/sqlancer/databend/DatabendSchema.java:344]: Perhaps `dataType.substring(0, dataType.indexOf(" "))` would be easier to understand?

**Review by mrigger — APPROVED** [review]
LGTM! Updating the branch before merging.

It seems that SQLancer found some (new?) logic bugs and an assertion failure in Databend according to the CI runs.

**Comment by hanyisong** [comment]
> LGTM! Updating the branch before merging.
> 
> It seems that SQLancer found some (new?) logic bugs and an assertion failure in Databend according to the CI runs.

Yes, I have submitted these bugs

**Comment by mrigger** [comment]
Would it be possible to ignore these bugs in https://github.com/sqlancer/sqlancer/blob/master/src/sqlancer/databend/DatabendBugs.java, so that the CI passes without errors until they are fixed?

**Pull request merged**